manageiq-operator/deploy/olm-catalog/manageiq-operator/0.0.1/manageiq.org_manageiqs_crd.yaml

```diff
@@ -51,11 +51,19 @@ spec:
             databaseVolumeCapacity:
               description: 'Database volume size (default: 15Gi)'
               type: string
+            httpdAuthenticationType:
+              description: 'Type of httpd authentication (default: internal) Options:
+                internal, external, active-directory, saml, openid-connect Note: external,
+                active-directory, and saml require an httpd container with elevated
+                privileges'
+              type: string
             httpdCpuRequest:
               description: 'Httpd deployment CPU request (default: no request)'
               type: string
             httpdImageName:
-              description: 'Image used for the httpd deployment (default: manageiq/httpd)'
+              description: 'Image namespace used for the httpd deployment (default:
```

Comment:
Since this is the namespace should the key be httpdImageNamespace?

Reply:
Yeah, missed a rebuild

Reply:
Fixed